src/parse_gtfs.py

```diff
@@ -167,6 +242,16 @@ def GenerateStops(gtfs):
   stops['lat'] = stops.geometry.y
   stops['lng'] = stops.geometry.x
 
+  #Drop unneeded columns
+  stops = stops.drop(columns=[
+    'wheelchair_boarding',
+    'stop_url',
+    'zone_id',
+    'stop_desc',
+    'location_type',
+    'stop_code'
+  ])
+
```

Comment:
Instead of dropping, I filter for cols I we want included: 

```python
  trips = trips[['route_id',
                'service_id',
                'trip_id',
                'trip_headsign',
                'block_id',
                'shape_id',
                'arrival_time',
                'departure_time',
                'stop_id',
                'timepoint'
               ]]
```

Reply:
This is a good idea. Please see revised PR.